packages/skeleton-svelte/src/lib/components/Tab/TabsControl.svelte

```diff
@@ -0,0 +1,126 @@
+<script lang="ts">
+	import { getContext } from 'svelte';
+	import type { TabsControlProps } from './types.js';
+
+	let {
+		name,
+		group,
+		title = '',
```

Comment:
The `title` prop feels out of place here. Let's maybe drop this. The ARAI attributes already provide enough metadata here.